lib/collection/proxy-config.js

```diff
@@ -181,7 +193,12 @@ _.assign(ProxyConfig.prototype, /** @lends ProxyConfig.prototype */ {
      * @returns {String}
      */
     getProxyUrl: function () {
-        return DEFAULT_PROTOCOL + PROTOCOL_HOST_SEPARATOR + this.host + COLON + this.port;
+        var proxyAuth = '',
+            auth = this.auth && this.auth.type === BASIC_AUTH_METHOD && this.auth.current();
```

Comment:
neatly put this in an if block below and do the assignment.

- single variable for auth string and call it `auth`